classes/Product.php

```diff
@@ -4057,7 +4057,8 @@ public static function getPriceFromOrder(
         int $combinationId,
         bool $withTaxes,
         bool $useReduction,
-        bool $withEcoTax
+        bool $withEcoTax,
+        int $ustomizationId = 0
```

Comment:
Missing a "c"

Reply:
You're right, it's the rush!
Thank you.